src/test/java/GoogleSearch.java

```diff
@@ -0,0 +1,37 @@
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.firefox.FirefoxDriver;
+
+import static java.lang.Thread.sleep;
+
+public class GoogleSearch {
+
+    private FirefoxDriver Driver;
```

Comment:
the convention for variables in Java is camelcase, for this case  change to `driver`

Reply:
Done